### ISSUE TITLE
Custom forms don't support collections

### ISSUE THREAD

**Issue by guidowb** [cf-platform-eng/tile-generator#117]
Or if the forms do, the values don't get passed to the apps in a usable way.

We have special handling for dynamic_service_plans, but we have a partner (Dynatrace) who wants two independent forms for two brokers in the same tile, and that can't be done today. If we implemented general collection handling, they could use that for multiple plan collections (and we may be able to eliminate most of the special handling for dynamic_service_plans)

**Comment by guidowb** [comment]
Form presentation has been fixed on a branch, but without special treatment of the `value`, the environment variable looks like this:

```
cf set-env app1-0.0.19 EXAMPLE_COLLECTION [#<OpenStruct guid="c1a8bf12-136d-4e15-9333-3dd28a330afc", album="Chronicles", artist="Rush", explicit=false>, #<OpenStruct guid="0de88737-bbce-408b-aff8-002c44655c3a", album="Greatest Hits", artist="Queen", explicit=true>]
```

Working on fixing that.

**Comment by guidowb** [comment]
Fixed by 27f58e6c599264adf4e24d7f53019e3dbc83f2e6
